proposals/0200-raw-string-escaping-revision-1.md

```diff
@@ -0,0 +1,454 @@
+# Enhancing String Literals Delimiters to Support Raw Text
+
+* Proposal: [SE-0200](0200-raw-string-escaping.md)
+* Author: [John Holdsworth](https://github.com/johnno1962), [Brent Royal-Gordon](https://github.com/brentdax), [Erica Sadun](https://github.com/erica)
+* Review Manager: [Doug Gregor](https://github.com/DougGregor)
+* Status: **Returned for revision**
+* Implementation: [apple/swift#17668](https://github.com/apple/swift/pull/17668)
+* Bugs: [SR-6362](https://bugs.swift.org/browse/SR-6362) **Needs Update for New Design**
+
```

Comment:
Could you add a "previous revision" field with a link to the current version of SE-0200?